lib/ansible/modules/cloud/azure/azure_rm_webapp_facts.py

```diff
@@ -92,58 +94,131 @@
         availability_state:
-            description: Availability of this web app.
+            description:
+                - Availability of this web app.
+            returned: always
             type: str
+            sample: Normal
         default_host_name:
-            description: Host name of the web app.
+            description:
+                - Host name of the web app.
+            returned: always
             type: str
+            sample: vxxisurg397winapp4.azurewebsites.net
         enabled:
-            description: Indicates the web app enabled or not.
+            description:
+                - Indicates the web app enabled or not.
+            returned: always
             type: bool
+            sample: true
         enabled_host_names:
-            description: Enabled host names of the web app.
+            description:
+                - Enabled host names of the web app.
+            returned: always
             type: list
+            sample: [
+                    "vxxisurg397winapp4.azurewebsites.net",
+                    "vxxisurg397winapp4.scm.azurewebsites.net"
+                    ]
         host_name_ssl_states:
-            description: SSL state per host names of the web app.
+            description:
+                - SSL state per host names of the web app.
+            returned: always
             type: list
+            sample: [
+                    {
+                        "hostType": "Standard",
+                        "name": "vxxisurg397winapp4.azurewebsites.net",
+                        "sslState": "Disabled"
+                    },
+                    {
+                        "hostType": "Repository",
+                        "name": "vxxisurg397winapp4.scm.azurewebsites.net",
+                        "sslState": "Disabled"
+                    }
+                    ]
         host_names:
-            description: Host names of the web app.
+            description:
+                - Host names of the web app.
+            returned: always
             type: list
+            sample: [
+                    "vxxisurg397winapp4.azurewebsites.net"
+                    ]
         outbound_ip_addresses:
-            description: Outbound ip address of the web app.
+            description:
+                - Outbound IP address of the web app.
+            returned: always
             type: str
+            sample: "40.71.11.131,40.85.166.200,168.62.166.67,137.135.126.248,137.135.121.45"
         ftp_publish_url:
-            description: Publishing url of the web app when deployment type is FTP.
+            description:
+                - Publishing url of the web app when deployment type is FTP.
+            returned: always
             type: str
             sample: ftp://xxxx.ftp.azurewebsites.windows.net
         state:
-            description: State of the web app.  eg. running.
+            description:
+                - State of the web app. for example C(running).
```

Comment:
Now that the entry has a `sample` line we can remove the example here.
```suggestion
                - State of the web app.
```